components/GitHubEventAction.tsx

```diff
@@ -63,6 +63,16 @@ function GitHubEventAction({type, payload}: GitHubEventActionProps): JSX.Element
       }
       return <span>{action} <ELink link={prURL}>pull request #{prNum}</ELink> in</span>;
     }
+    case 'PullRequestReviewCommentEvent': {
+      const action = payload.action;
+      const prNum = payload.comment.number;
+      const prLink = payload.comment.pull_request_url;
+      return (
+        <span>
+          {action} <ELink link={prLink}> pull request #{prNum}</ELink> in
+        </span>
```

Comment:
Do you mind keeping the spacing/newlines consistent with the other code?

Reply:
Yea, sorry about that. Couldn't get my user.workspace in VSCode to not apply ESlint in this workspace. Should be fixed now.